src/validation/CampaignSchema.tsx

```diff
@@ -104,16 +104,30 @@ export const CampaignSchema = (prices: AdvertiserPriceFragment[]) =>
         then: (schema) =>
           findPrice(
             prices,
-            CampaignFormat.NewsDisplayAd,
+            [CampaignFormat.NewsDisplayAd],
             BillingType.Cpm,
             "10",
             schema,
           ),
       })
+      .when(["billingType", "format"], {
+        is: (b: string, f: CampaignFormat) =>
+          b === "cpqv" &&
+          (f === CampaignFormat.NewsDisplayAd ||
+            CampaignFormat.PushNotification),
+        then: (schema) =>
+          findPrice(
+            prices,
+            [CampaignFormat.NewsDisplayAd, CampaignFormat.PushNotification],
+            BillingType.Cpsv,
+            "3",
+            schema,
+          ),
+      })
       .required("Price is a required field"),
     billingType: string()
       .label("Pricing Type")
-      .oneOf(["cpm", "cpc"])
+      .oneOf(["cpm", "cpc", "cpsv"])
       .required("Pricing type is a required field"),
     adSets: array()
```

Comment:
We need some tests around this schema, please. This is too complex to validate behaviour by reading.